yb-voyager/cmd/importData.go

```diff
@@ -527,7 +527,13 @@ func applyTableListFilter(importFileTasks []*ImportFileTask) []*ImportFileTask {
 	if len(allUnknown) > 0 {
 		utils.PrintAndLog("Unknown table names in the table-list: %v", allUnknown)
 		utils.PrintAndLog("Valid table names are: %v", lo.Map(allTables, func(t sqlname.NameTuple, _ int) string {
-			return t.ForOutput()
+			if t.TargetTableAvailable() {
```

Comment:
Can you do a code scan to identify if there can be potentially other bugs similar to this pattern?